go/kbfs/libkbfs/tlf_journal_test.go

```diff
@@ -532,6 +532,7 @@ func testTLFJournalBlockOpDiskByteLimit(t *testing.T, ver kbfsmd.MetadataVer) {
 	md := config.makeMD(kbfsmd.RevisionInitial, kbfsmd.ID{})
 	err = tlfJournal.doOnMDFlushAndRemoveFlushedMDEntry(
 		ctx, kbfsmd.ID{}, &RootMetadataSigned{RootMetadataSigned: kbfsmd.RootMetadataSigned{MD: md.bareMd}})
+	require.NoError(t, err)
```

Comment:
@strib is an error expected here? https://ci.keyba.se/job/client/job/PR-17681/4/execution/node/2354/log/

Reply:
I took a quick look and yeah it seems like an error is expected here, it's doing something kinda funny for test purposes.

Reply:
thx!